packages/cli/src/commands/apps/favorites/add.ts

```diff
@@ -0,0 +1,44 @@
+import color from '@heroku-cli/color'
+import {ux} from '@oclif/core'
+import {Command, flags} from '@heroku-cli/command'
+import {Favorites} from '../../../lib/types/favortites'
```

Comment:
Update file name from `favortites` to `favorites`

Reply:
Nice catch